packages/typography/src/elements/Code.spec.tsx

```diff
@@ -45,30 +44,4 @@ describe('Code', () => {
       expect(container.firstChild).toHaveStyleRule('font-size', '17px');
     });
   });
-
-  describe('hue', () => {
-    it('renders grey hue if provided', () => {
-      const { container } = render(<Code hue="grey" />);
-
-      expect(container.firstChild).toHaveStyleRule('background-color', PALETTE_V8.grey[200]);
-    });
-
-    it('renders green hue if provided', () => {
-      const { container } = render(<Code hue="green" />);
-
-      expect(container.firstChild).toHaveStyleRule('background-color', PALETTE_V8.green[200]);
-    });
-
-    it('renders red hue if provided', () => {
-      const { container } = render(<Code hue="red" />);
-
-      expect(container.firstChild).toHaveStyleRule('background-color', PALETTE_V8.red[200]);
-    });
-
-    it('renders yellow hue if provided', () => {
-      const { container } = render(<Code hue="yellow" />);
-
-      expect(container.firstChild).toHaveStyleRule('background-color', PALETTE_V8.yellow[200]);
-    });
-  });
 });
```

Comment:
Moved specs to `StyledCode.spec.tsx`.

Reply:
Thank you for clarifying. I wasn't sure about that. I'll move the specs back to this file instead.